enhancements/platform-validation.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Kepler Platform Validation
+authors:
+  - Jie Ren (jie.ren@intel.com)
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2023-05-27
+last-updated: 2023-05-27
+tracking-links: # link to related GitHub issues
+  - TBD
+---
+
+# Kepler Platform Validation Guide
+
+## Summary
+
+A quick start guide to validate Kepler support on specific hardware platform.
+
+## Motivation
+
+Currently, Kepler's test cases focus on code logic verification, lack of actual platform validation.
+
+Kepler's [data](https://github.com/sustainable-computing-io/kepler/tree/main/data) files majorly contain static information, or lack of data sync mechanism.
+
+Test cases and test report are not available, which could be the evidence or check criteria to below concerns:
+
+1. Whether the specific CPU/GPU/FPGA/Accelerator model is well supported in Kepler?
+
+2. Whether the data/metrics of specific power source are correctly exported?
+
+3. Whether the power measurement accuracy is accpetable?
```

Comment:
what's the difference with the 2nd item?

Reply:
The 2nd item focus on the physical correctness of those metrics collected from specific power source, i.e. RAPL. But most of such metrics are node level or CPU PKG level.
The 3rd item focus on the fairness, and check either ratio based power estimation or model based power estimation is reasonable, acceptable, accurate, especially on Dynamic power estimation. Which is more attractive to customers who care about their workload's carbon footprint than customers who own the whole node/the whole cluster, such as CSP. 

> what's the difference with the 2nd item?

The 2nd item focus on the physical correctness of those metrics collected from specific power source, i.e. RAPL. But most of such metrics are node level or CPU PKG level.
The 3rd item focus on the fairness, and check either ratio based power estimation or model based power estimation is reasonable, acceptable, accurate, especially on Dynamic power estimation. Which is more attractive to customers who care about their workload's carbon footprint than customers who own the whole node/the whole cluster, such as CSP.